modules/exploits/multi/ssh/sshexec.rb

```diff
@@ -109,17 +109,18 @@ def initialize
               'Arch'     => ARCH_PYTHON,
               'Platform' => 'python'
             }
+          ],
+          [
+            'Unix Cmd',
+            {
+              'Arch'     => ARCH_CMD,
+              'Platform' => 'unix'
+            }
           ]
         ],
       'DefaultTarget'    => 0,
       # For the CVE
-      'DisclosureDate'   => 'Jan 01 1999',
-      'Notes'            =>
```

Comment:
It looks like these `notes` were removed in error. Can you double check that you forked this branch from the latest `master` ?

Reply:
Just added them back. The notes section was made about 10 days ago and I never merged them.